test/utils/getDefaultCommitContent.spec.js

```diff
@@ -0,0 +1,34 @@
+import fs from 'fs'
+import getDefaultCommitContent from '../../src/utils/getDefaultCommitContent'
+
+describe('getDefaultCommitContent', () => {
+  describe('when file exists', () => {
+    beforeAll(() => {
+      process.argv[3] = 'commit'
```

Comment:
It's not dangerous to overwrite the `process.argv` inside of tests? 🤔

Reply:
Since we mock the `fs` functions that use the value of `process.argv[3]`, I think we can just not bother ovrewriting it at all